bin/bench-prover/src/main.rs

```diff
@@ -0,0 +1,45 @@
+use std::path::PathBuf;
+
+use bench_prover::{
+    benchmark_names::{BENCH_CONSUME_MULTIPLE_NOTES, BENCH_CONSUME_NOTE_NEW_ACCOUNT, BENCH_GROUP},
+    utils::{cargo_target_directory, process_benchmark_data, save_json_to_file},
+};
+use serde_json::json;
+
+fn main() -> std::io::Result<()> {
+    let target_dir = cargo_target_directory().unwrap_or_else(|| PathBuf::from("target"));
+    let base_path = target_dir.join("criterion").join(BENCH_GROUP);
+
+    println!("Looking for benchmark results in: {}", base_path.display());
+
+    let benchmarks = vec![BENCH_CONSUME_NOTE_NEW_ACCOUNT, BENCH_CONSUME_MULTIPLE_NOTES];
```

Comment:
```suggestion
    let benchmarks = [BENCH_CONSUME_NOTE_NEW_ACCOUNT, BENCH_CONSUME_MULTIPLE_NOTES];
```

Nit: Could just be an array.